### PR TITLE
adds new brain damage lines

### PR DESCRIPTION
**Pull request by thestubborn**
edit as needed, im new to all this, so bare with my retardtation, adds such gems as "hog" and "latex vox are cute"

### CODE REVIEW

**Comment by DrSnips** [comment]
I'd rather not have sexual ones like latex vox, cooming, and penis hurt. Personal preference aside while latex vox has some actually noteworthy history with the server the other two are just a bit too general.

**Comment by thestubborn** [comment]
> 
> 
> I'd rather not have sexual ones like latex vox, cooming, and penis hurt. Personal preference aside while latex vox has some actually noteworthy history with the server the other two are just a bit too general.

cooming and penis hurt i'd be 100% ok with removing, just addded em because they were from a ban  i found funny, latex vox however, deserves it out of sheer infamy imo

**Comment by N3X15** [comment]
All these sorts of lists need to be made files, like the name lists.

**Comment by boy2mantwicethefam** [comment]
love it

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm [hunk 1]
```diff
@@ -79,10 +79,17 @@
 						"why woud i take a pin pointner??", \
 						"FUCK IT; KISSYOUR ASSES GOOD BYE DEAD MEN! I AM SELFDESTRUCKTING THE STATION!!!!", \
 						"How do I set up the. SHow do I set u p the Singu. how I the scrungularity????", \
-						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!"))
+						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!", \
+						"Latex vox are cute!", \
+						"HOG", \
+						";ITS LOOSE", \
+						"I C-CUM-M-M-MM-M-MED-D-D!!", \
```
Comment (jknpj) [inline code/modules/mob/living/carbon/human/life/handle_disabilities.dm:86]: No

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm [hunk 2]
```diff
@@ -79,10 +79,17 @@
...
+						"VOX ARE POX", \
+						"my body is on the asteroi", \
+						";how do i plasmaflood", \
+						"M-M-M-MY-Y P-P-P-PENIS-S IS HURTIN-N-NG-G-G!"))
```
Comment (jknpj) [inline code/modules/mob/living/carbon/human/life/handle_disabilities.dm:90]: NO

Comment (MadmanMartian) [inline code/modules/mob/living/carbon/human/life/handle_disabilities.dm:90]: No

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm [hunk 3]
```diff
@@ -79,10 +79,17 @@
...
 						"How do I set up the. SHow do I set u p the Singu. how I the scrungularity????", \
-						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!"))
+						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!", \
+						"Latex vox are cute!", \
+						"HOG", \
+						";ITS LOOSE", \
+						"I C-CUM-M-M-MM-M-MED-D-D!!", \
+						"VOX ARE POX", \
+						"my body is on the asteroi", \
+						";how do i plasmaflood", \
```
Comment (MadmanMartian) [inline code/modules/mob/living/carbon/human/life/handle_disabilities.dm:89]: No banbaiting for braindamage lines, please

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm [hunk 4]
```diff
@@ -79,10 +79,17 @@
...
 						"FUCK IT; KISSYOUR ASSES GOOD BYE DEAD MEN! I AM SELFDESTRUCKTING THE STATION!!!!", \
 						"How do I set up the. SHow do I set u p the Singu. how I the scrungularity????", \
-						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!"))
+						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!", \
+						"Latex vox are cute!", \
+						"HOG", \
+						";ITS LOOSE", \
+						"I C-CUM-M-M-MM-M-MED-D-D!!", \
+						"VOX ARE POX", \
+						"my body is on the asteroi", \
```
Comment (MadmanMartian) [inline code/modules/mob/living/carbon/human/life/handle_disabilities.dm:88]: No banbaiting for braindamage lines, please

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm [hunk 5]
```diff
@@ -79,10 +79,17 @@
 						"why woud i take a pin pointner??", \
 						"FUCK IT; KISSYOUR ASSES GOOD BYE DEAD MEN! I AM SELFDESTRUCKTING THE STATION!!!!", \
 						"How do I set up the. SHow do I set u p the Singu. how I the scrungularity????", \
-						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!"))
+						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!", \
+						"Latex vox are cute!", \
```
Comment (MadmanMartian) [inline code/modules/mob/living/carbon/human/life/handle_disabilities.dm:83]: What the fuck

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm [hunk 6]
```diff
@@ -79,10 +79,17 @@
...
+						"HOG", \
+						";ITS LOOSE", \
```
Comment (MadmanMartian) [inline code/modules/mob/living/carbon/human/life/handle_disabilities.dm:85]: No shuttlecall baiting for braindamage lines, please

**Comment by SonixApache** [comment]
"no"

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm (hunk 4 above)
Comment (DrSnips) [reply code/modules/mob/living/carbon/human/life/handle_disabilities.dm:88]: This one actually has some pretty good historical significance. The lack of a D  indicates it being a reference.

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm (hunk 5 above)
Comment (thestubborn) [reply code/modules/mob/living/carbon/human/life/handle_disabilities.dm:83]: latex vox is a classic

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm (hunk 6 above)
Comment (thestubborn) [reply code/modules/mob/living/carbon/human/life/handle_disabilities.dm:85]: ehhh, id be ok with removing the ;

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm (hunk 4 above)
Comment (thestubborn) [reply code/modules/mob/living/carbon/human/life/handle_disabilities.dm:88]: if an admin bans an alive person for saying their body is on the asteroid, they'd be a pretty dumb admin

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm (hunk 1 above)
Comment (thestubborn) [reply code/modules/mob/living/carbon/human/life/handle_disabilities.dm:86]: did say i was ok w/ removing that one

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm (hunk 2 above)
Comment (thestubborn) [reply code/modules/mob/living/carbon/human/life/handle_disabilities.dm:90]: ditto

**Comment by thestubborn** [comment]
smh

**Pull request closed without merging**